.eslintrc.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   root: true,
   env: {
     node: true,
+    "vue/setup-compiler-macros": true,
```

Comment:
What does this?

Reply:
With the new setup script, you can use defineProps without defining it. With an old vue-eslint-parser, this generates an no-undef error wich is disabled with this line.